### PR TITLE
RDoc-2114-Created 5.1 Server/config/licencing folder and overviw doc, updated studio doc

### PR DESCRIPTION
**Pull request by serezhub**
…

### CODE REVIEW

File: Documentation/5.1/Raven.Documentation.Pages/studio/server/license-management.markdown [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+![Force Update](images/manage-license-3.png "Force Update")
+
+{NOTE License information is automatically updated nightly.  _FORCE UPDATE_ does not stop this operation. /}
+
+{PANEL/}
+
+{PANEL: Renew License}
+
+If your server is connected to the automatic update [License Server](../../studio/server/license-management#license-information),
```
Comment (grisha-kotler) [inline Documentation/5.1/Raven.Documentation.Pages/studio/server/license-management.markdown:121]: This isn't correct.
The Renew button is only available for Community and Developer licenses.
The license can only be renewed when less than 30 days are left for the expiration.
The Community license will be extended by 1 year.
The Developer license will be extended by 6 months.

File: Documentation/5.1/Raven.Documentation.Pages/studio/server/license-management.markdown [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+your license will automatically update as long as it hasn't been canceled.
+
+If you aren't connected to the License Server, copy the entire license code block from the license renewal email and paste it into
```
Comment (grisha-kotler) [inline Documentation/5.1/Raven.Documentation.Pages/studio/server/license-management.markdown:124]:
```suggestion
If you aren't connected to the License Server, copy the entire license key from the license renewal email and paste it into 
```

File: Documentation/5.1/Raven.Documentation.Pages/server/configuration/licensing/licensing-overview.markdown [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+* There are four licensing options available for you to choose from according to your needs.  
+  See the [pricing page](https://ravendb.net/buy) for more info about which features are included in each license.  
+   * **Development** (fully featured and temporary - must be upgraded before launch)
+   * **Community** (basic production-grade)
+   * **Professional** (standard production-grade)
+   * **Enterprise** (high-performance, fully-featured production-grade)
+   * [Cloud portal](../../../cloud/cloud-overview) for cloud-based servers
+
+* Your license code is sent to the email address that you entered when registering your license.
```
[the remark this answers is not in the document]
Comment (serezhub) [reply Documentation/5.1/Raven.Documentation.Pages/server/configuration/licensing/licensing-overview.markdown:14]: done

File: Documentation/5.1/Raven.Documentation.Pages/server/configuration/licensing/licensing-overview.markdown [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+   * **Enterprise** (high-performance, fully-featured production-grade)
+   * [Cloud portal](../../../cloud/cloud-overview) for cloud-based servers
+
+* Your license code is sent to the email address that you entered when registering your license.  
+  It is in .json format.  
+
+* Licenses are activated and updated automatically unless: 
+  * You change the [license configurations](../../../server/configuration/licensing/license-configuration) in your server's settings.json.
+  * RavenDB's "License Server" cannot connect to check and update any changes in your license.  
+    This is usually either caused by a firewall blocking the connection or when servers run offline.
```
[the remark this answers is not in the document]
Comment (serezhub) [reply Documentation/5.1/Raven.Documentation.Pages/server/configuration/licensing/licensing-overview.markdown:20]: done

File: Documentation/5.1/Raven.Documentation.Pages/server/configuration/licensing/licensing-overview.markdown [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+
+* Your license code is sent to the email address that you entered when registering your license.  
+  It is in .json format.  
+
+* Licenses are activated and updated automatically unless: 
+  * You change the [license configurations](../../../server/configuration/licensing/license-configuration) in your server's settings.json.
+  * RavenDB's "License Server" cannot connect to check and update any changes in your license.  
+    This is usually either caused by a firewall blocking the connection or when servers run offline.  
+    * See Studio [License Server](../../../studio/server/license-management#license-information) to check your connection to
+      RavenDB's automatic License Server.
```
[the remark this answers is not in the document]
Comment (serezhub) [reply Documentation/5.1/Raven.Documentation.Pages/server/configuration/licensing/licensing-overview.markdown:22]: done

File: Documentation/5.1/Raven.Documentation.Pages/server/configuration/licensing/licensing-overview.markdown [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+
+* Licenses can also be [managed via Studio](../../../studio/server/license-management).
+
+In this page:
+
+* [Activating Your License](../../../server/configuration/licensing/licensing-overview#activating-your-license)
+* [Changing Your License](../../../server/configuration/licensing/licensing-overview#changing-your-license)
+* [Renewing Your License](../../../server/configuration/licensing/licensing-overview#renewing-your-license)
+* [Force Update for Immediate Usage](../../../server/configuration/licensing/licensing-overview#force-update-for-immediate-usage)
```
[the remark this answers is not in the document]
Comment (serezhub) [reply Documentation/5.1/Raven.Documentation.Pages/server/configuration/licensing/licensing-overview.markdown:31]: done

File: Documentation/5.1/Raven.Documentation.Pages/server/configuration/licensing/licensing-overview.markdown [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+---
+
+{PANEL: Activating Your License}
+
+Because licenses are activated and updated automatically by default, the following sections are only relevant if you 
+changed the license configurations in the server's [settings.json](../../../server/configuration/configuration-options#json)
+or if your server is running offline.
+
+To activate your license, you must either embed the code directly into the settings.json, 
+or provide a path to the license code as a .json file.
```
[the remark this answers is not in the document]
Comment (serezhub) [reply Documentation/5.1/Raven.Documentation.Pages/server/configuration/licensing/licensing-overview.markdown:45]: done

File: Documentation/5.1/Raven.Documentation.Pages/server/configuration/licensing/licensing-overview.markdown [hunk 8]
```diff
@@ -0,0 +1,146 @@
...
+
+To activate your license, you must either embed the code directly into the settings.json, 
+or provide a path to the license code as a .json file.  
+
+You can either [activate via Studio](../../../studio/server/license-management#register-license) or use the following 
+[configuration](../../../server/configuration/licensing/license-configuration) options in settings.json:
+
+* **License**  
+    Embed the full license string for RavenDB. If `License` is specified, it overrides the `License.Path` configuration.  
+    E.g. - `"License": paste your license code including curly brackets here`
```
[the remark this answers is not in the document]
Comment (serezhub) [reply Documentation/5.1/Raven.Documentation.Pages/server/configuration/licensing/licensing-overview.markdown:52]: done

File: Documentation/5.1/Raven.Documentation.Pages/server/configuration/licensing/licensing-overview.markdown [hunk 9]
```diff
@@ -0,0 +1,146 @@
...
+To activate your license, you must either embed the code directly into the settings.json, 
+or provide a path to the license code as a .json file.  
+
+You can either [activate via Studio](../../../studio/server/license-management#register-license) or use the following 
+[configuration](../../../server/configuration/licensing/license-configuration) options in settings.json:
+
+* **License**  
+    Embed the full license string for RavenDB. If `License` is specified, it overrides the `License.Path` configuration.  
+    E.g. - `"License": paste your license code including curly brackets here`  
+    Your license code was sent to the email that is registered with the license.
```
[the remark this answers is not in the document]
Comment (serezhub) [reply Documentation/5.1/Raven.Documentation.Pages/server/configuration/licensing/licensing-overview.markdown:53]: done

File: Documentation/5.1/Raven.Documentation.Pages/server/configuration/licensing/licensing-overview.markdown [hunk 10]
```diff
@@ -0,0 +1,146 @@
...
+    with machine performance specs similar to your plans for production. This way you can securely test the API and processing of 
+    data that mimics your actual data-set and planned infrastructure.  
+
+{NOTE/}
+
+{PANEL/}
+
+{PANEL: Changing Your License}
+
+Upon [changing your license](https://ravendb.net/buy), RavenDB will send the new license code to the email
```
[the remark this answers is not in the document]
Comment (serezhub) [reply Documentation/5.1/Raven.Documentation.Pages/server/configuration/licensing/licensing-overview.markdown:78]: done in all places where "code" should be "key"

File: Documentation/5.1/Raven.Documentation.Pages/server/configuration/licensing/licensing-overview.markdown [hunk 11]
```diff
@@ -0,0 +1,146 @@
...
+Upon [changing your license](https://ravendb.net/buy), RavenDB will send the new license code to the email 
+registered with the license.  
+Copy the entire code block and either:  
+
+* [Update via Studio](../../../studio/server/license-management#replace-license) 
+* [Replace the license code](../../../server/configuration/licensing/licensing-overview#activating-your-license) where it is located (either directly in the settings.json or 
+  in the file path that settings.json is configured for.)  
+
+If the default connection with RavenDB's [License Server](../../../studio/server/license-management#license-information) is active, 
+the license will update tonight.
```
[the remark this answers is not in the document]
Comment (serezhub) [reply Documentation/5.1/Raven.Documentation.Pages/server/configuration/licensing/licensing-overview.markdown:87]: done

File: Documentation/5.1/Raven.Documentation.Pages/server/configuration/licensing/licensing-overview.markdown [hunk 12]
```diff
@@ -0,0 +1,146 @@
...
+
+By default, RavenDB automatically renews a license as long as it isn't canceled.  
+If the automatic renewal feature is turned off in the settings.json, you can renew your license with the same process 
+as [activating your license](../../../server/configuration/licensing/licensing-overview#activating-your-license).
+
+{PANEL/}
+
+{PANEL: Force Update for Immediate Usage}
+
+Unless the feature is disabled, RavenDB automatically checks and implements license updates every night.
```
[the remark this answers is not in the document]
Comment (serezhub) [reply Documentation/5.1/Raven.Documentation.Pages/server/configuration/licensing/licensing-overview.markdown:109]: done

File: Documentation/5.1/Raven.Documentation.Pages/studio/server/license-management.markdown (hunk 2 above)
Comment (serezhub) [reply Documentation/5.1/Raven.Documentation.Pages/studio/server/license-management.markdown:124]: I replaced code/code block with key everywhere in the document.

**Comment by Danielle9897** [comment]
Closing this PR.
New PR is: https://github.com/ravendb/docs/pull/1595

**Pull request closed without merging**